src/main/java/br/com/invillia/lyon/userevents/mapper/UserMapper.java

```diff
@@ -0,0 +1,24 @@
+package br.com.invillia.lyon.userevents.mapper;
+
+import br.com.invillia.lyon.userapi.events.UserEvent;
+import br.com.invillia.lyon.userevents.domain.User;
+import br.com.invillia.lyon.userevents.response.UserResponse;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+
+    public User updateUser(UserResponse userResponse, String id) {
```

Comment:
Não parece que esse método vai atualizar o user. O que esse método está fazendo?

Reply:
Ele tem a responsabilidade de setar os valores do UserResponse para o User

Reply:
Então qual seria um bom nome pra ele?

Reply:
setting Values ​​for User ?